modules/server/src/network/httpResponses.ts

```diff
@@ -26,9 +26,9 @@ export type Result<T, FailureStatus extends string, FailureData = void> =
  * @returns {boolean} Whether the Result was a Success or not
  */
 export function isSuccess<T, FailureStatus extends string, FailureData>(
-	result: Result<T, FailureStatus, FailureData>,
+	result?: Result<T, FailureStatus, FailureData>,
 ): result is Success<T> {
-	return result.status === 'SUCCESS';
+	return result !== undefined && result.status === 'SUCCESS';
```

Comment:
We should probably have 'SUCCESS' as a constant somewhere.

Reply:
After reviewing where this was called... We really shouldnt have the situation where we have `Result<> | undefined`, that goes against the entire purpose of returning that type. We probably should revert this change.